pom.xml

```diff
@@ -17,7 +17,7 @@
         <jenkins.version>1.651.3</jenkins.version>
         <java.level>7</java.level>
 
-        <azure-commons-version>0.2.4</azure-commons-version>
+        <azure-commons-version>0.2.6</azure-commons-version>
```

Comment:
Waiting for the deployment of azure-commons 0.2.6. Tested locally with SNAPSHOT version.

Reply:
azure-commons 0.2.6 released. trigger the build check